justfile

```diff
@@ -78,6 +78,10 @@ android args="":
 android-release:
     cd mobile/native && cargo ndk -t armeabi-v7a -t arm64-v8a -o ../android/app/src/main/jniLibs build --release --verbose
 
+# Build flutter webapp for cargo run --bin webapp
```

Comment:
❓ Shall we add another recipe for `cargo run --bin webapp` then? Or combine them i.e. `just run-web`.